src/App.js

```diff
@@ -1,16 +1,82 @@
 import React from 'react';
+import TodoForm from "./components/TodoComponents/TodoForm";
+import TodoList from "./components/TodoComponents/TodoList";
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super();
+    this.state = {
+      manyTodo: [],
+      task: "",
+      completed: false,
+    };
+  }
+
+  inputHander = (event) => {
+    const value = event.target.value;
+    this.setState({
+      task: value
+    })
+    //console.log(value)
+  };
+
+  submitHander = (event) => {
+    event.preventDefault();
+    const singleTodo = {
+      task: this.state.task,
+      id: Date.now(),
+      completed: this.state.completed,
+    };
+    this.setState({
+      manyTodo:[...this.state.manyTodo, singleTodo,],
+      task: ""
+    });
+    console.log("blank")
+    console.log(event.target.value)
+    
+  }
+
+  finishedHander = (todoId) =>{
+    this.setState({
+      manyTodo: this.state.manyTodo.map(todo => {
+        if(todo.id === todoId){
+          return {
+            task: todo.task,
+            id: todo.id,
+            completed: !todo.completed
+          } 
+          }else {
+            return todo
+        }
+      })
+    })
+  }
+
+  clearHander = (event) =>{
+    event.preventDefault();
+    this.setState({
+      manyTodo: this.state.manyTodo.filter(item =>{
+        if(item.completed === false){
+          return{
+            manyToDo: item
+          }
+        }
+      })
+    })
+    
+
+  }
+
   render() {
     return (
       <div>
-        <h2>Welcome to your Todo App!</h2>
+        <h2>Get Er' Done</h2>
+        <TodoList finishedHander ={this.finishedHander} manyTodo={this.state.manyTodo} />
+        <TodoForm inputHander={this.inputHander} submitHander ={this.submitHander} clearHander={this.clearHander} />
```

Comment:
I noticed in your TodoForm you are calling `props.value` but you are never passing the `value={this.state.task}` here